cmd/create/operatorroles/cmd.go

```diff
@@ -126,15 +135,15 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	if len(missingRoles) == 0 &&
-		cluster.State() != cmv1.ClusterStateWaiting && cluster.State() != cmv1.ClusterStatePending {
+		cluster.State() != cmv1.ClusterStateWaiting && cluster.State() != cmv1.ClusterStatePending &&
+		!args.forcePolicyCreation {
```

Comment:
Why do we need to check the force flag here?

Reply:
They might have all roles but not up to date on the policies